tests/phpunit/tests/blocks/register.php

```diff
@@ -305,6 +321,28 @@ public function test_success_register_block_style_handle() {
 		);
 	}
 
+	/**
+	 * @ticket 55513
+	 */
+	public function test_success_register_block_style_handle_in_theme() {
+		switch_theme( 'block-theme' );
+
+		$metadata = array(
+			'file'  => DIR_TESTDATA . '/themedir1/block-theme/blocks/example-block/block.json',
+			'name'  => 'block-theme/example-block',
+			'style' => 'file:./style.css',
+		);
+		$result   = register_block_style_handle( $metadata, 'style' );
```

Comment:
So this one tres to register the `style` but the theme reigsters the `style` in `block.json` and both use the same filename, right?